tools/nmr_annotation2d/annotationRmn2D_xml.xml

```diff
@@ -223,24 +232,29 @@
     </outputs>
     <tests>
     	<test>

```

Comment:
please add `expect_num_outputs` https://docs.galaxyproject.org/en/latest/dev/schema.html#id92

This will test the output filters as well.